docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -352,6 +354,19 @@ branch as soon as possible (if they weren't cherry-picked from it). Avoid
 merging the tagged release into the branch to achieve this, as this will include
 the release commit, which may pin Composer dependencies to specific versions.
 
+#### Updating Composer requirements in minor releases
+
+We keep core modules in lockstep at the minor level - that is, we can release
+patches (e.g. 4.5.x) for individual modules, but when we perform a minor release
+(4.x.0), we ship that version of every core module. To this end, the Composer
+dependencies of each module need to be manually adjusted when we perform a minor
+release - for example, the `cms` module version `4.6.0` must include a minimum
+requirement of `framework` `^4.6`. This ensures that language level requirements
+(e.g. minimum PHP versions) can be safely centralised in the framework module
+for surrounding core modules to inherit. In short, ensure you commit updates to
+the Composer requirements of every core module after each minor branch is
+created, and before you ship the release.
+
```

Comment:
since it's not documented anywhere else, would it be worth listing out here what the core modules are?
otherwise, maybe make a page on silverstripe.org list them?
- silverstripe/admin
- silverstripe/assets
- silverstripe/asset-admin
- silverstripe/campaign-admin
- silverstripe/cms
- silverstripe/config
- silverstripe/errorpage
- silverstripe/framework
- silverstripe/reports
- silverstripe/siteconfig
- silverstripe/versioned
- silverstripe/versioned-admin

Reply:
Quick note: there's a manually constructed list here: https://github.com/silverstripeltd/github-issue-search-client/blob/master/src/repos.json which you might want to put somewhere more centralised like here: https://github.com/silverstripe/supported-modules/blob/gh-pages/modules.json, in order to auto-generate that list. It will change over time, an example is that login-forms is now a core module, and versioned-snapshots might become one soon.

Reply:
Good idea.  Have raised an issue to implement https://github.com/silverstripe/supported-modules/issues/10